index.js

```diff
@@ -427,6 +446,31 @@ class Jarvis {
       return true;
     }
   }
+
+  /**
+   * Load a provided env file
+   * Save env variables as constants
+   */
+  async _loadEnv(envFile) {
+    let fileContent;
+    try {
+      fileContent = parseScript(envFile);
+    } catch (error) {
+      throw new Error('Could not read env file from specified location!');
+    }
+    if (fileContent[0] === "in this context" && fileContent[fileContent.length - 1] === "end") {
```

Comment:
a. convert to guard clause and return early
b. since meaning of the line is not immediately obvious, add line comment